target/linux/ramips/rt305x/base-files/etc/board.d/02_network

```diff
@@ -201,6 +202,7 @@ ramips_setup_macs()
 	poray,ip2202|\
 	teltonika,rut5xx|\
 	unbranded,xdx-rn502j|\
+	zyxel,keenetic-lite_rev.b|\
```

Comment:
Is there an appropriate address in 0x2e?
Does the device have a label with a MAC address on it and if yes, to which interface does it correspond?

Reply:
deleted this line

Reply:
Why? And what's with my question?

Reply:
Does `rt2800pci-phy0::radio` work? I've never seen it on RT5350 before.

Reply:
Unfortunately I do not know the answer to this question, I just adapt the old patch for the new version of openwrt
https://github.com/trexxet/openwrt-zyxel_keenetic/blob/master/0019-ramips-add-ZyXel-Keenetic-userspace.patch

Reply:
Just run the following and put the result here:
```
. /lib/functions/system.sh
. /lib/functions.sh
mtd_get_mac_binary factory 0x4
mtd_get_mac_binary factory 0x28
mtd_get_mac_binary factory 0x2e
mtd_get_mac_binary factory 0x8004
mtd_get_mac_binary factory 0xe000
mtd_get_mac_binary factory 0xe006
```

Only the last byte of the addresses is relevant, so you might just hide the rest of the returned addresses if you want to, e.g.
xx:xx:xx:xx:xx:34

And please do the same for the addresses on the device's outside label, if it has one.

Reply:
`root@OpenWrt:~# . /lib/functions/system.sh`
`root@OpenWrt:~# . /lib/functions.sh`
`root@OpenWrt:~# mtd_get_mac_binary factory 0x4`
`xx:xx:xx:xx:xx:5f`
`root@OpenWrt:~# mtd_get_mac_binary factory 0x28`
`xx:xx:xx:xx:xx:60`
`root@OpenWrt:~# mtd_get_mac_binary factory 0x2e`
`ff:ff:ff:ff:ff:ff`
`root@OpenWrt:~# mtd_get_mac_binary factory 0x8004`
`ff:ff:ff:ff:ff:ff`
`root@OpenWrt:~# mtd_get_mac_binary factory 0xe000`
`ff:ff:ff:ff:ff:ff`
`root@OpenWrt:~# mtd_get_mac_binary factory 0xe006`
`ff:ff:ff:ff:ff:ff`
>And please do the same for the addresses on the device's outside label, if it has one.

Sorry, the case of the router has long been thrown away.

Reply:
[Googling](https://60.img.avito.st/1280x960/8128330760.jpg) shows this model has WAN MAC address on its label

Reply:
Yes it does, but I only have a bare board. The case on which the label was located I threw away a long time ago. If this is so important, I can ask the people on the forum to execute these commands and send me a response and the photo of label.